### PR TITLE
Add Cooperative Delivery WG Charter

### PR DESCRIPTION
**Pull request by thschue**
Signed-off-by: Thomas Schuetz <thomas.schuetz@dynatrace.com>

### CODE REVIEW

File: cooperative-delivery-wg/README.md
Comment (hongchaodeng) [inline cooperative-delivery-wg/README.md:6]: Shall we decide a meeting schedule before merging this PR?

File: cooperative-delivery-wg/charter.md
Comment (hongchaodeng) [inline cooperative-delivery-wg/charter.md:3]: Maybe move this section to the end? Otherwise it is too long to put at the beginning...

File: cooperative-delivery-wg/README.md
Comment (thschue) [reply cooperative-delivery-wg/README.md:6]: I think we should decide this in one of the first meetings ...

**Review by AlexsJones — APPROVED** [review]

File: cooperative-delivery-wg/charter.md
Comment (joshgav) [inline cooperative-delivery-wg/charter.md:99]: +1

File: cooperative-delivery-wg/charter.md
Comment (joshgav) [inline cooperative-delivery-wg/charter.md:105]: s/micros-service/microservice/

**Review by joshgav — APPROVED** [review]
LGTM, thanks @thschue. You included an interesting list of application delivery patterns, I commented some thoughts for your consideration but no need to worry too much. I've been thinking about putting together some sort of capability and style matrix for the delivery tools in podtato-head.

File: cooperative-delivery-wg/charter.md
Comment (roberthstrand) [reply cooperative-delivery-wg/charter.md:71]: Both Helm and Kustomize is used within GitOps so this description is very accurate. There are essentially nothing different about how the applications is deployed, besides the fact that it's not you who run the tools but it's operators.

File: cooperative-delivery-wg/charter.md
Comment (thschue) [reply cooperative-delivery-wg/charter.md]: Done

File: cooperative-delivery-wg/charter.md
Comment (thschue) [reply cooperative-delivery-wg/charter.md]: Done

**Review by AloisReitbauer — APPROVED** [review]

**Pull request merged**
